packages/components/src/components/domainproperties/issues/IssuesListDefDesignerPanels.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { List } from 'immutable';
+import produce, { Draft } from 'immer';
+
+import { BaseDomainDesigner, InjectedBaseDomainDesignerProps, withBaseDomainDesigner } from '../BaseDomainDesigner';
+import { getDomainPanelStatus, saveDomain } from '../actions';
+import { resolveErrorMessage } from '../../../util/messaging';
+import DomainForm from '../DomainForm';
+import { DomainDesign } from '../models';
+
+import { IssuesListDefPropertiesPanel } from './IssuesListDefPropertiesPanel';
+import { IssuesListDefModel } from './models';
+
+interface Props {
+    initModel?: IssuesListDefModel;
+    onChange?: (model: IssuesListDefModel) => void;
+    onCancel: () => void;
+    onComplete: (model: IssuesListDefModel) => void;
+    useTheme?: boolean;
+    containerTop?: number; // This sets the top of the sticky header, default is 0
+    successBsStyle?: string;
+    saveBtnText?: string;
+}
+
+interface State {
+    model: IssuesListDefModel;
+}
+
+class IssuesDesignerPanelsImpl extends React.PureComponent<Props & InjectedBaseDomainDesignerProps, State> {
+    constructor(props: Props & InjectedBaseDomainDesignerProps) {
+        super(props);
+
+        this.state = {
+            model: props.initModel || IssuesListDefModel.create({}),
```

Comment:
Nick reminded me that we should also be using `produce` here in the constructor so that our initial state object is truly immutable. Should be a simple change, see my recent change in the constructor for `DataClassDesigner`.

Reply:
Good to know! Now changed. Also have made a similar change in couple other places.